plugins/httpapi/dcnm.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2023 Cisco and/or its affiliates.
+# Copyright (c) 2020-2025 Cisco and/or its affiliates.
```

Comment:
NOTE: I would not suggest reviewing the diffs but rather compare the code here to what we have in the `develop` branch